### DISCUSSION TITLE
Optimization in webpack

### DISCUSSION THREAD

**Discussion by evenstensberg** [nasa/openmct#3937] [Community Q&A]
Hi!

I was wondering if OpenMCT could add dev/prod builds and optimizations to css and js? I can happily help with this

**Comment by jvigliotta** [comment]
hey @evenstensberg, I'm going to move this over to our Discussions pages, I think it's better suited there.

**Reply by jvigliotta** [reply]
going to alert @unlikelyzero, @charlesh88, @nikhilmandlik as well to get more eyes.

**Reply by nikhilmandlik** [reply]
currently WIP to upgrade to webpack5 #4242

**Comment by akhenry** [comment]
@akhenry

**Reply by evenstensberg** [reply]
Anything you need from webpack side of things at the moment?
